medical-portal/src/UI/src/app/case-details/case-details.component.html

```diff
@@ -97,7 +97,11 @@ <h2>Active Case Details</h2>
         orientation="horizontal"
       >
         <mat-step state="circle">
-          <ng-template matStepLabel>Opened:<br /></ng-template>
+          <ng-template matStepLabel
+            >Opened:<br />{{
+              caseDetails?.openedDate | date: "longDate"
+            }}</ng-template
+          >
```

Comment:
you don't need to put the `>` on a new line here

Reply:
I tried moving  > to the top but when i save that it auto formats and add that in new line

Reply:
VS Code is probably using your .editorconfig. You can edit these changes, you can also disable auto formatting. I try to not be opinionated on formatting but this one in particular is hard to read. You will want to change that sooner than later